### PR TITLE
TASK SIO-3732 Add virtualization types for CR settings

### PR DESCRIPTION
**Pull request by rincler**

### CODE REVIEW

File: computeResourcesSettings_test.go [hunk 1]
```diff
@@ -11,13 +11,14 @@ import (
...
-		Network:         ComputeResourceSettingsNetwork{},
-		BalanceStrategy: ComputeResourceBalanceStrategyMostStorageAvailable,
+		CachePath:           "fake cache path",
+		ISOPath:             "fake ISO path",
+		BackupTmpPath:       "fake backup tmp path",
+		VNCProxyPort:        1337,
+		Limits:              ComputeResourceSettingsLimits{},
+		Network:             ComputeResourceSettingsNetwork{},
+		BalanceStrategy:     ComputeResourceBalanceStrategyMostStorageAvailable,
+		VirtualizationTypes: []VirtualizationTypeName{},
```
Comment (dshemin) [inline computeResourcesSettings_test.go:21]: Please put some values here, in order to make sure we send them.

File: computeResource.go [hunk 2]
```diff
@@ -84,13 +84,14 @@ const (
...
-	Network         ComputeResourceSettingsNetwork `json:"network"`
-	BalanceStrategy ComputeResourceBalanceStrategy `json:"balance_strategy"`
+	CachePath           string                         `json:"cache_path"`
+	ISOPath             string                         `json:"iso_path"`
+	BackupTmpPath       string                         `json:"backup_tmp_path"`
+	VNCProxyPort        int                            `json:"vnc_proxy_port"`
+	Limits              ComputeResourceSettingsLimits  `json:"limits"`
+	Network             ComputeResourceSettingsNetwork `json:"network"`
+	BalanceStrategy     ComputeResourceBalanceStrategy `json:"balance_strategy"`
+	VirtualizationTypes []VirtualizationTypeName       `json:"virtualization_types"`
```
Comment (dshemin) [inline computeResource.go:94]: According to current design of this structure I can send next data to CR's setting update endpoint:
```
data := ComputeResourceSettings{
  ...
  VirtualizationTypes: []VirtualizationTypeName{
   VirtualizationTypeKVM,
   VirtualizationTypeKVM,
   VirtualizationTypeKVM,
   ...
  }
}
```
Is it okay? Did our API handle this case?

**Review by dshemin — CHANGES_REQUESTED** [review]

**Review by oneumyvakin — APPROVED** [review]

File: computeResource.go (hunk 2 above)
Comment (rincler) [reply computeResource.go:94]: API filters duplicate values now

File: computeResourcesSettings_test.go (hunk 1 above)
Comment (rincler) [reply computeResourcesSettings_test.go]: Done

**Review by dshemin — APPROVED** [review]

**Pull request merged**